src/org/zaproxy/zap/extension/jython/ZapAddOn.xml

```diff
@@ -5,7 +5,7 @@
 	<description>Allows Python to be used for ZAP scripting - templates included</description>
 	<author>ZAP Dev Team</author>
 	<url></url>
-	<changes>Minor code changes</changes>
+	<changes>Add HTTP Sender template script.</changes>
```

Comment:
Removed the previous change as that's no longer needed IMHO, the previous change was just added to not leave the changes empty after bumping the version (I don't think that needs to be noted in the changes other than in this case). If you think it's better leave it I'll add it again.